bears/general/CLOCBear.py

```diff
@@ -0,0 +1,63 @@
+import re
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='cloc',
+        use_stdout=True,
+        use_stderr=False)
+class CLOCBear:
+    """
+    Summarises file of or number of files in directory structure
+    with total lines, number comment lines, actual code lines using tool cloc
+    """
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='cloc')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Number of comment lines', 'Number of Empty lines'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return ('', filename)
+
+    def process_output(self, output, filename, file):
+        out = output
+        lines = out.split('\n')
+        files = (int)(re.search(r'(\d+)', lines[1]).group(1))
```

Comment:
Not familiar with this syntax :/ this is typecasting syntax in c, if you are looking for typecasting use ` int(...)`